testament/testament.nim

```diff
@@ -1196,7 +1433,7 @@ proc parseArgs(execState: var Execution, p: var OptParser): ParseCliResult =
     execState.filter = TestFilter(kind: tfkAll)
   of "c", "cat", "category":
     # only specified category
-    # xxx: consider removing pcat concept or make parallel the default
+    # HACK consider removing pcat concept or make parallel the default
```

Comment:
It's a consideration or something to think about not a hack, the semantics shifted slightly with megatest gone.

Reply:
Ok, I just fixed the woefully undescriptive `xxx:` tags and just misinterpreted this one a bit.

Reply:
Sometimes the vagueness of xxx is an asset, often a curse.